Code/Editor/Core/EditorActionsHandler.cpp

```diff
@@ -141,6 +188,186 @@ void EditorActionsHandler::InitializeActions()
         );
     }
 
+    // --- Game Actions
+
+    // Play Game
+    {
+        AzToolsFramework::ActionProperties actionProperties;
+        actionProperties.m_name = "Play Game";
+        actionProperties.m_description = "Activate the game input mode.";
+        actionProperties.m_category = "Game";
+        actionProperties.m_iconPath = ":/stylesheet/img/UI20/toolbar/Play.svg";
+
+        m_actionManagerInterface->RegisterCheckableAction(
+            EditorMainWindowActionContextIdentifier, "o3de.action.game.play", actionProperties,
+            [cryEdit = m_cryEditApp]()
+            {
+                cryEdit->OnViewSwitchToGame();
+            },
+            []()
+            {
+                return GetIEditor()->IsInGameMode();
+            }
+        );
```

Comment:
For customizable toolbars, we're gonna need to store these in json files so that's definitely coming. That will probably trickle back to system toolbars down the line.

About the context and action identifiers, keeping them in the function signature is deliberate since they are required parameters, while properties inside ActionProperties are mostly not required.